internal/cmd/beta/security-group/create/create.go

```diff
@@ -0,0 +1,180 @@
+package create
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/goccy/go-yaml"
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+)
+
+const (
+	nameFlag        = "name"
+	descriptionFlag = "description"
+	statefulFlag    = "stateful"
+	labelsFlag      = "labels"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	Labels      *map[string]any
+	Description string
+	Name        string
+	Stateful    bool
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates security groups",
+		Long:  "Creates security groups.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(`Create a named group`, `$ stackit beta security-group create --name my-new-group`),
+			examples.NewExample(`Create a named group with labels`, `$ stackit beta security-group create --name my-new-group --labels label1=value1,label2=value2`),
+		),
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to create the security group %q?", model.Name)
+				err = p.PromptForConfirmation(prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			request := buildRequest(ctx, model, apiClient)
+
+			operationState := "Created"
+			p.Info("%s security group %q for %q\n", operationState, model.Name, model.ProjectId)
+
+			group, err := request.Execute()
+			if err != nil {
+				return fmt.Errorf("create security group: %w", err)
+			}
+			if err := outputResult(p, model, group); err != nil {
+				return err
+			}
+
+			return nil
+		},
+	}
+
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().String(nameFlag, "", "the name of the security group. Must be <= 63 chars")
+	cmd.Flags().String(descriptionFlag, "", "an optional description of the security group. Must be <= 127 chars")
+	cmd.Flags().Bool(statefulFlag, false, "create a stateful or a stateless security group")
+	cmd.Flags().StringSlice(labelsFlag, nil, "Labels are key-value string pairs which can be attached to a network-interface. E.g. '--labels key1=value1,key2=value2,...'")
```

Comment:
There are some flag descriptions still non-capitalized

Reply:
Capitalized correctly in all commands